library/Ivoz/Core/Infrastructure/Domain/Service/Cgrates/CompanyBalanceService.php

```diff
@@ -19,6 +19,42 @@ public function __construct(
         $this->client = $client;
     }
 
+    /**
+     * Simulate call and get price
+     *
+     * @param string $tenant
+     * @param string $subject
+     * @param int $durationSeconds
+     */
+    public function simulateCall(string $tenant, string $subject, int $durationSeconds)
+    {
+        $answerDateTime = new \DateTime(
+            time(),
+            new \DateTimeZone('UTC')
+        );
+
+        $payload = [
+            'Tenant' => $tenant,
+            'Subject' => $subject,
+            'Category' => 'call',
+            'AnswerTime' => $answerDateTime->format('Y-m-dTH:i:sZ'),
```

Comment:
The other changes on this commit use as data format `Y-m-d\TH:i:s\Z`, while this uses `Y-m-dTH:i:sZ` not sure if this is a problem or not :relaxed: